NitroxClient/Communication/Packets/Processors/RocketResyncProcessor.cs

```diff
@@ -0,0 +1,64 @@
+﻿using NitroxClient.Communication.Packets.Processors.Abstract;
+using NitroxClient.MonoBehaviours;
+using NitroxClient.Unity.Helper;
+using NitroxModel_Subnautica.Packets;
+using UnityEngine;
+
+namespace NitroxClient.Communication.Packets.Processors;
+
+public class RocketResyncProcessor : ClientPacketProcessor<RocketResync>
+{
+    public override void Process(RocketResync packet)
+    {
+        GameObject gameObjectRocket = NitroxEntity.RequireObjectFrom(packet.RocketId);
+
+        gameObjectRocket.RequireComponentInChildren<RocketPreflightCheckManager>().preflightChecks.Clear();
+        RocketPreflightCheckScreenElement[] screenElements = gameObjectRocket.GetComponentsInChildren<RocketPreflightCheckScreenElement>(true);
+        CockpitSwitch[] cockpitSwitches = gameObjectRocket.GetComponentsInChildren<CockpitSwitch>(true);
+        ThrowSwitch[] throwSwitches = gameObjectRocket.GetComponentsInChildren<ThrowSwitch>(true);
+
+        // TODO: REMOVE BEFORE MERGE
```

Comment:
These are the steps to test the resync of the preflight checks
Just a note to remove before merging